test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -467,7 +467,7 @@ protected <A extends InternalAggregation, C extends Aggregator> A searchAndReduc
         throws IOException {
         IndexSettings indexSettings = createIndexSettings();
         // First run it to find circuit breaker leaks on the aggregator
-        runWithCrankyCircuitBreaker(indexSettings, searcher, aggTestConfig);
+        // runWithCrankyCircuitBreaker(indexSettings, searcher, aggTestConfig);
```

Comment:
I guess this is a left over?

Reply:
Good catch. I repeatedly commented that out and uncommented it in order to make debugging easier, but clearly forgot to uncomment it the last time!